.gitignore

```diff
@@ -5,4 +5,10 @@ hc.sqlite
 static-collected
 
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+hc/blogs/tests/test_blog_frontend.py
+geckodriver
+geckodriver.log
+hc/blogs/geckodriver
+hc/blogs/tests/chromedriver
+hc/blogs/tests/geckodriver
```

Comment:
why are these here?

Reply:
I was avoiding to commit these in git. I think I need to remove them from the file and stop tracking them on git